package.json

```diff
@@ -19,6 +19,7 @@
     "gatsby-transformer-sharp": "^1.6.22",
     "gulp-sass": "^4.0.2",
     "lodash": "^4.17.15",
+    "react-mailchimp-subscribe": "^2.1.0",
```

Comment:
This isnt necessary, but to prevent issues down the line we may want to lock all dependency versions. So instead of `"lodash": "^4.17.15"` it would be `"lodash": "4.17.15"`